README.md

```diff
@@ -1,62 +1,181 @@
 
-Run the following line-by-line.
+It is built using [Docker](https://www.docker.com/), which you need to install, but it does not require any real Docker knowledge to use.
+
+Our aim is to get you to a working setup within 10-minutes from now (presuming you have a decent internet connection).
+It's also possible to get each piece of Exercism working on your local machine without this repository, but that will take lots more effort and work.
+
+## Basic setup
+
+The following instructions take you through getting the most basic setup working.
+We'll explain what actually happens below the hood, and how to configure things afterwards
+
+### Getting set up
+
+To start, run the following instructions line-by-line:
 
 ```bash
 # Make a directory to host all of Exercism within
 mkdir exercism
 
-# cd into it
+# Move into the new directory
 cd exercism
 
-# Clone this repo
+# Clone this repository onto your computer
 git clone git@github.com:exercism/v3-docker-compose.git
 
-# Clone any essential repos you plan to hack on
-git clone git@github.com:exercism/v3-website.git
-git clone git@github.com:exercism/tooling-orchestrator.git
-git clone git@github.com:exercism/tooling-invoker.git
-git clone git@github.com:exercism/csharp-test-runner.git
-
+# Move into the new directory
 cd v3-docker-compose
 
-# start with the default stack
+# Create your "stack" - the collection of parts of Exercism you
+# want to run locally. For now just copy the default file.
 cp stack.default.yml stack.yml
-# edit stack.yml to enabled the services (runners, analyzers, you'll be using)
-./bin/build_stack
 
-# Build everything in parallel
-docker-compose build --parallel
+# Start everything
+./bin/start
+```
+
+### Updating everything
+
+The first time you use `./bin/start` we will download the latest versions of all the Exercism components for you to use.
+When you want to upgrade those components, run:
+
+```bash
+docker-compose pull
+```
+
+If you face any issues getting started, we recommend running this step in case anything is cached locally from some previous installation.
+
+### Logs
+
+Once the docker stack is running you can tail the logs using:
 
-# This is your normal starting command
-docker-compose up --build
+```bash
+docker-compose logs -f
+```
+
+You can also tail just one component by specifying its name.
+For example, to tail the website logs:
+
+```bash
+docker-compose logs -f website
 ```
 
-## Hacking on certain components of the architecture
+### Shelling into a container
+
+From time to time you may want to shell into a running container to run commands or check what is going on.
+To shell into a running container you can use the provided `bin/shell` command.
+For example, to shell into the running `website` container, you would run:
+
+```bash
+bin/shell website
+```
+
+### Stopping everything
+
+To stop everything, run:
+```
+docker-compose down
+```
+
```

Comment:
Instruction to stop/restart a single thing?

Reply:
@SleeplessByte Currently the restarting bit is below when discussing how to actually hack on things. Do you think it should move here instead?

Reply:
Yeh, or at least "how to restart listed below". I expected it to be here.